rpcs3/keyboard_pad_handler.cpp

```diff
@@ -60,9 +60,40 @@ void keyboard_pad_handler::keyPressEvent(QKeyEvent* event)
 		event->ignore();
 		return;
 	}
-
-	Key(event->key(), 1);
-	event->ignore();
+	switch (event->key())
+	{
+		case Qt::Key_L:
+			if (!(event->modifiers() == Qt::AltModifier)) { Key(event->key(), 1); }
+			event->ignore();
+			break;
+		case Qt::Key_Return:
+			if (!(event->modifiers() == Qt::AltModifier)) { Key(event->key(), 1); }
+			event->ignore();
+			break;
+		case Qt::Key_Escape:
+			event->ignore();
+			break;
+		case Qt::Key_P:
+			if (!(event->modifiers() == Qt::ControlModifier)) { Key(event->key(), 1); }
+			event->ignore();
+			break;
+		case Qt::Key_S:
+			if (!(event->modifiers() == Qt::ControlModifier)) { Key(event->key(), 1); }
+			event->ignore();
+			break;
+		case Qt::Key_R:
+			if (!(event->modifiers() == Qt::ControlModifier)) { Key(event->key(), 1); }
+			event->ignore();
+			break;
+		case Qt::Key_E:
+			if (!(event->modifiers() == Qt::ControlModifier)) { Key(event->key(), 1); }
+			event->ignore();
+			break;
+		default:
+			Key(event->key(), 1);
+			event->ignore();
```

Comment:
I didn't really look into it yet but shouldn't default just break?

Reply:
I just went with the same behaviour that was there before. I can check if break would work.

Reply:
oh sorry, I somehow missed the old code on my phone. nvm then